core/src/test/java/com/linecorp/armeria/client/proxy/ProxyConfigTest.java

```diff
@@ -44,19 +44,6 @@ void testEqualityAndHashCode(ProxyConfig config) {
         assertThat(equalProxyConfigs.get(0).hashCode()).isEqualTo(config.hashCode());
     }
 
-    @Test
-    void testUnresolvedProxyAddress() {
```

Comment:
Should we revive this test and fix it to check if `ProxyConfig` is successfully created with unresolved addresses?

Reply:
Thanks for your comments 🙇‍♂️ 
Good point, I didn't expect it needed to be kept at all. 😓
I revived this test and fixed it!